charts/estafette-k8s-hpa-scaler/templates/clusterrole.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: {{.Chart.Name | quote}}
+  labels:
```

Comment:
should we add these labels to every resource?

Reply:
I'd say yes. I see most of public charts have them on all resources, so I assume it's best practice. But I don't think it's critical we can add them later at any time.

Reply:
In the Estafette GKE extension I set the same labels everywhere, including on statefulset pvc template. Only in podspec I add some extra labels, like released version, who released it, etc. So in this Helm chart I would make sure you have the same set of labels everywhere and then the possibility to add some extra - like the version - in the podspec. But perhaps things like the version are useful in more places? What do you think?

Reply:
I believe we need the chart version everywhere, it can be in the form suggested by @lruslan `{{ .Chart.Name }}-{{ .Chart.Version }}`.
And then inside podspec we can add the actual version of image (`{{ .Chart.AppVersion }}`).